exercises/stack_03.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+// Stack.push implementation
+(function(){
+
+    function Stack(arr) {
+        this.arr = arr;
+        return;
+    }
+
+    Stack.prototype.forEach = function(callback) {
+        for (var i = 0; i < this.arr.length; i++) {
+            callback.call(this, this.arr[i], i, this.arr);
+        }
+
+        return;
+    }
+
+    Stack.prototype.push = function(item) {
+        this.arr.push(item);
+
+        return;
+    }
+
+    Stack.prototype.peek = function() {
+        return this.arr[0];
```

Comment:
This actually peaks at the first element of the "Stack", but we intend for `peek` to peek at the last element (the last push'ed element). Can you change it to that?